eZ/Publish/Core/FieldType/RichText/Resources/schemas/docbook/docbook.iso.sch.xsl

```diff
@@ -79,4 +79,7 @@
 <axsl:template match="db:link" priority="1000" mode="M3"><svrl:fired-rule xmlns:svrl="http://purl.oclc.org/dsdl/svrl" context="db:link"/>
 
 		<!--ASSERT -->
-<axsl:choose><axsl:when test="not(contains(@*[name()='xlink:href'], 'javascript:') or contains(@*[name()='xlink:href'], 'vbscript:'))"/><axsl:otherwise><svrl:failed-assert xmlns:svrl="http://purl.oclc.org/dsdl/svrl" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:schold="http://www.ascc.net/xml/schematron" test="not(contains(@*[name()='xlink:href'], 'javascript:') or contains(@*[name()='xlink:href'], 'vbscript:'))"><axsl:attribute name="location"><axsl:apply-templates select="." mode="schematron-get-full-path-2"/></axsl:attribute><svrl:text>using scripts in links is not allowed</svrl:text></svrl:failed-assert></axsl:otherwise></axsl:choose><axsl:apply-templates select="*|comment()|processing-instruction()" mode="M3"/></axsl:template><axsl:template match="text()" priority="-1" mode="M3"/><axsl:template match="@*|node()" priority="-2" mode="M3"><axsl:apply-templates select="*|comment()|processing-instruction()" mode="M3"/></axsl:template></axsl:stylesheet>
+<axsl:choose><axsl:when test="not(contains(@*[name()='xlink:href'], 'javascript:') or contains(@*[name()='xlink:href'], 'vbscript:'))"/><axsl:otherwise><svrl:failed-assert xmlns:svrl="http://purl.oclc.org/dsdl/svrl" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:schold="http://www.ascc.net/xml/schematron" test="not(contains(@*[name()='xlink:href'], 'javascript:') or contains(@*[name()='xlink:href'], 'vbscript:'))"><axsl:attribute name="location"><axsl:apply-templates select="." mode="schematron-get-full-path-2"/></axsl:attribute><svrl:text>using scripts in links is not allowed</svrl:text></svrl:failed-assert></axsl:otherwise></axsl:choose>
```

Comment:
Note to reviewer: Those two validators overlaps. Should we remove the first one as more specific?

Reply:
referring to the `axsl:when`  vs the `svrl:failed-assert` on this line?

Or in relevance to line below?

Reply:
I'm referring to those selected lines https://github.com/adamwojs/ezpublish-kernel/blob/bedace3e674301cb60dff863b94528ee2cc7620c/eZ/Publish/Core/FieldType/RichText/Resources/schemas/docbook/docbook.iso.sch#L255-L256  

_I should be more precise, sorry for that_

Reply:
Ok, maybe @alongosz has input on this?

Reply:
The only one side effect of using both of them at once double validation error:
![zrzut ekranu 2017-10-02 o 16 37 48](https://user-images.githubusercontent.com/211967/31082831-3bea7084-a790-11e7-9ad7-a701c7a0eec3.png)

Reply:
ah, we don't want that.. could you find a agreement with @alongosz ? @pspanja is back from vacation so might be possible to reach as well ;)

Reply:
AFAICS full error msg of your validator here is `/section/para[2]/link: The value must be a valid URL`, right? If that's the case, then it's fine to remove my validator, it's ugly :-)
BTW. How does it look like in the case of multiple invalid links in multiple paragraphs?

But please see this general remark I posted some time ago:
> General remark: `docbook.iso.sch.xsl` is generated from `docbook.iso.sch` and the latter one should be changed (Sorry for the inconvenience!). 
See [README](https://github.com/ezsystems/ezpublish-kernel/blob/v6.11.2/eZ/Publish/Core/FieldType/RichText/Resources/schemas/docbook/Readme.rst#ez-publish-docbook-variant) to find out how to generate it.

So, in short, it cannot be delivered in the current form.
Two directions:
1. Generate this XSL using schematron file as described above.
2. If it's impossible to produce the samle xsl using schematron, then maybe we should abandon schematron and change file suffix from `.sch.xsl` to just `.xsl`

Reply:
Please disregard part about schematron, I didn't notice source file .sch is also a part of this PR. Now it's fine.